pysb_model.py

```diff
@@ -0,0 +1,26 @@
+# exported from PySB model 'None'
```

Comment:
Not sure how this file showed up here (got generated by the PysbAssembler at some point) but we don't need it in version control.